src/components/signoff/SimpleReview/PerRecordDiffView.tsx

```diff
@@ -160,6 +166,34 @@ function DiffLabel({ changeType }: { changeType: ChangeType }) {
   }
 }
 
+export function formatDiffHeader({
+  source,
+  target,
+  displayFields = [],
+}: {
+  source: ValidRecord;
+  target: ValidRecord;
+  displayFields: string[];
+}) {
+  let fields = [];
+
+  for (let f of displayFields) {
+    fields.push(
+      <span>
+        <label>{f}:</label> {(target || source)[f] || "undefined"}
```

Comment:
Since we support dotted field names to target nested objects (eg. `"book.author.name"`), we should use [`renderDisplayField()`](https://github.com/Kinto/kinto-admin/blob/f201c095c81331f295ac988227bbb88446f3f364/src/utils.tsx#L196C17-L196C35) here.

Reply:
Oh, hadn't considered nested properties being displayed and I didn't write code to compensate for that. Updating to use `renderDisplayField` which also takes care of the `undefined` issue in a consistent way.